src/main/java/io/wispforest/worldmesher/WorldMesh.java

```diff
@@ -351,6 +360,11 @@ public Builder disableCulling() {
             return this;
         }
 
+        public Builder enableGlobalNeighbors() {
```

Comment:
Given that you already call the backing field `useGlobalNeighbors`, I think calling this method the same would be ideal. After that, we're ready for merge